hathor/indexes/rocksdb_address_index.py

```diff
@@ -148,11 +148,14 @@ def get_sorted_from_address(self, address: str) -> List[bytes]:
     def is_address_empty(self, address: str) -> bool:
         self.log.debug('seek to', address=address)
         it = self._db.iterkeys(self._cf)
-        it.seek(self._to_key(address))
-        res = it.get()
-        if not res:
+        key = self._to_key(address)
+        it.seek(key)
+        cf_key = it.get()
+        if not cf_key:
+            return True
+        if self._db.get(cf_key) is None:
```

Comment:
Why don't we do `key == cf_key` instead of hitting the database? I'd also add a note explaining that the `it.get()` returns the seek key when it reaches the end of the database.

Reply:
That makes sense, I changed the method to use that check instead of a `db.get`.

Reply:
Great! Thanks!